tools/proteomiqon_mzmltomzlite/proteomiqon_mzmltomzlite.xml

```diff
@@ -0,0 +1,292 @@
+                                        "Fields" :
+                                        [
+                                            {
+                                                #if $Ms2PeakPickingCond.CentroidModeCond.PaddingParamsCond.MaximumPaddingPointsCond.specifyMaximumPaddingPoints
+                                                "MaximumPaddingPoints" : {
+                                                    "Case" : "Some",
+                                                    "Fields" : [
+                                                        $Ms2PeakPickingCond.CentroidModeCond.PaddingParamsCond.MaximumPaddingPointsCond.MaximumPaddingPoints
+                                                    ]
+                                                },
+                                                #else
+                                                "MaximumPaddingPoints" : null,
+                                                #end if
+                                                "Padding_MzTolerance" : $Ms2PeakPickingCond.CentroidModeCond.PaddingParamsCond.Padding_MzTolerance,
+                                                "WindowSize" : $Ms2PeakPickingCond.CentroidModeCond.PaddingParamsCond.WindowSize,
+                                                "SpacingPerc" : $Ms2PeakPickingCond.CentroidModeCond.PaddingParamsCond.SpacingPerc
+                                            }
+                                        ]
+
+                                    }
+                                    #else
+                                    "PaddingParams" : null
+                                    #end if
+                                }
+                               
+                            ]
+                            #end if
+                        }
+                    ]
+#end if
+                }
+            }
+        </configfile>
+    </configfiles>
+    <inputs>
+        <param type="data" name="instrumentOutput" format="mzml" label="Instrument output" help="Please specify the .mzML file you want to convert"/>
+        <param name="BinaryDataCompressionType" type="select" label="Binary data compression type">
+            <option value="0" selected="true">No compression</option>
+            <option value="1">ZLib</option>
+            <option value="2">Num press</option>
+            <option value="3">Num press ZLib</option>
+            <option value="4">Num press Pic</option>
+            <option value="5">Num press Lin</option>
+        </param>
+        <conditional name="StartRetentionTimeCond">
+            <param type="boolean" name="useStartRetentionTime" label="Use start retention time" checked="false" help="You can optionally specify a minimum scan time. MS scans measured earlier will not be copied."/>
+            <when value="true">
+                <param type="float" name="StartRetentionTime" label="Start retention time" value="0"/>
```

Comment:
if you just want to make this variable optional you can add "optional="true" and remove the entire conditional